domains/hymoonz.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "hymoonz.is-a.dev",
+    "repo": "https://github.com/lrmn7/hymoonz.github.io",
+    "owner": {
+        "username": "hymoonz",
+        "email": "bilal190620@gmail.com"
+    },
+    "record": {
+        "CNAME": "hymoonz.github.io"
```

Comment:
Invalid record, just redirects back to this domain which you are requesting, which would then redirect back to your domain, which creates a redirect loop. This is invalid.